docs/en/engines/table-engines/special/buffer.md

```diff
@@ -8,6 +8,10 @@ sidebar_label:  Buffer
 
 Buffers the data to write in RAM, periodically flushing it to another table. During the read operation, data is read from the buffer and the other table simultaneously.
 
+:::note
+To enable asynchronous inserts, use the [async_insert](../../../operations/settings/settings.md#async-insert) setting.
```

Comment:
```suggestion
A recommended alternative to the Buffer Table Engine is using [asynchronous inserts](/docs/en/cloud/bestpractices/asyncinserts.md).
```

Reply:
Once https://github.com/ClickHouse/clickhouse-docs/pull/629 is merged, then this one can be updated with the URL under Optimize and merged.